packages/ndla-ui/src/Notion/ConceptNotion.tsx

```diff
@@ -28,15 +28,19 @@ export interface ConceptNotionType {
 }
 interface Props {
   concept: ConceptNotionType;
+  disableScripts?: boolean;
 }
 
-const ConceptNotion = ({ concept }: Props) => {
+const ConceptNotion = ({ concept, disableScripts }: Props) => {
   const notionId = `notion-${concept.id}`;
   const figureId = `notion-figure-${concept.id}`;
   const visualElementId = `visual-element-${concept.id}`;
 
   useEffect(() => {
-    initArticleScripts();
+    if (!disableScripts) {
+      initArticleScripts();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Hvorfor vil vi ikke ha `disableScripts` i dependency lista istedet for å skru av linten?

Reply:
Godt poeng. Jeg la det til i dependencies